build/dependencies.props

```diff
@@ -15,6 +15,13 @@
     <SystemTextJson>8.0.5</SystemTextJson>
   </PropertyGroup>
 
+<PropertyGroup Condition="'$(TargetFramework)' == 'net10.0'">
+    <!--
+      Unknown if 10.0 has the same problem as 9 where there are some runtime incompatibilities if 
```

Comment:
The issue is we want to use a LTS target, 9.0 is not long term, which is the reason CoreWCF asked us to stick with 8.0.

Is 10 LTS?